utils/pickImage.ts

```diff
@@ -0,0 +1,19 @@
+import * as ImagePicker from 'expo-image-picker';
+
+//The result object contains information about the selected image, such as the URI of the image file.
+const pickImage = async () => { 
+    let result = await ImagePicker.launchImageLibraryAsync({
+        mediaTypes: ImagePicker.MediaTypeOptions.Images,
+        allowsEditing: true,
+        aspect: [4, 3],
+        quality: 1,
+    });
+
+    console.log(result);
```

Comment:
Is this for testing purposes, or is it because the backend isn't set up?

Reply:
To see what a image is composed of for future delvelopment (type like png/jpeg, size, location ect.)